### DISCUSSION TITLE
Humanity of the move

### DISCUSSION THREAD

**Discussion by 01epa** [official-stockfish/Stockfish#4503] [Ideas]
Greetings. It is common knowledge that computer programs have long been stronger than humans. Very often the computer suggests moves that are very difficult even for a grandmaster. The likelihood of such a move is simply minimal. Is it possible to add a calculation of such a probability to the engine?
Do not ask how to do this, but I think it would be very revealing.

Thanks.

**Comment by PGG106** [comment]
this is either impossible or if possible would divert a lot of time and resources from actually making the engine stronger, it might be a fun project for a solo dev but there's no chance the actual sf team works on this

**Comment by dubslow** [comment]
It's certainly possible in the modern world of neural nets, but it's somewhat separate from the goal of engines, which is simply playing strength. Rather it would be a standalone project, kinda related to ChessCoach, tho different from that too.

https://github.com/chrisbutner/ChessCoach
